include/zephyr/bluetooth/hci_types.h

```diff
@@ -3547,7 +3547,7 @@ struct bt_hci_evt_le_cs_read_remote_supported_capabilities_complete {
 struct bt_hci_evt_le_cs_read_remote_fae_table_complete {
 	uint8_t status;
 	uint16_t conn_handle;
-	uint8_t remote_fae_table[72];
+	int8_t remote_fae_table[72];
```

Comment:
This means we require all Zephyr ABIs' signed number representation to be two's complement. I'm fine with this. This it true for all modern architectures, AFAIK. Maybe it would be best if this was a documented requirement. @jhedberg, what do you think? Would it be possible to get it documented officially?

Reply:
Just FYI: We already have 40 signed int values in this file, so this is nothing new :)

Reply:
Yeah, not sure if this needs to be called out separately, but it's certainly not anything new that this PR is introducing.